solutions/16.js

```diff
@@ -1,4 +1,28 @@
-//Manik Sachdeva: msach22
+// Rahul Kalra
+// should retirn true if the input has duplicates values and false if it doesn't
+
+const solution = (array) => {
+  let data = {};
+  for(let i = 0; i < array.length; i++){
+    if(!(data[array[i]])){
+      data[array[i]] = 1;
+    }
+    else{
```

Comment:
if data exist, that means its been found before. you can just return true here and finish the loop